exercises/hamming/src/test/java/HammingTest.java

```diff
@@ -43,14 +43,14 @@ public void testSmallHammingDistanceInLongerStrand() {
 
     @Ignore
     @Test
-    public void testValidatesFirstStrandNotLonger() {
+    public void testValidatesFirstStrandNotLonger() throws IllegalArgumentException {
```

Comment:
You don't need to add this exception to the signature as it is an [unchecked exception](https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html) :)